resources/metadata/detectExecuteScan.yaml

```diff
@@ -570,3 +590,40 @@ spec:
       options:
         - name: -u
           value: "0"
+      volumeMounts:
+        - mountPath: /opt/blackduck/blackduck-imageinspector/shared
+          name: imageinspector-shared
```

Comment:
please change the name to 
`name: volume`  since an earlier change introduced required the volume name to be strict names and only allow empty names to be mounted into the container 

the same comment for all the side car volume mount as well

Reply:
@vijayanjay will you adapt this PR or should we?